implement-basic-throttle.js

```diff
@@ -0,0 +1,44 @@
+// This is a JavaScript coding problem from BFE.dev 
+
+/**
+ * @param {(...args:any[]) => any} func
+ * @param {number} wait
+ * @returns {(...args:any[]) => any}
+ */
+
+function throttle(func, wait) {
```

Comment:
I should leave a comment from our chat. I think it's a good solution for tech interview, despite the bug with 3 and 4 console logs.